internal/defence/limiter_test.go

```diff
@@ -0,0 +1,80 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package defence
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	"storj.io/storj/internal/testcontext"
+)
+
```

Comment:
Also let's add an example for usage.

Reply:
example of usage is added in description of PR. Should i make more descriptive readme file?

Reply:
https://blog.golang.org/examples